packages/sources/layer2-sequencer-health/src/config/index.ts

```diff
@@ -46,14 +51,30 @@ export const HEALTH_ENDPOINTS = {
     endpoint: util.getEnv('METIS_HEALTH_ENDPOINT') || DEFAULT_METIS_HEALTH_ENDPOINT,
     responsePath: ['healthy'],
   },
+  [Networks.Starkware]: {
+    endpoint: util.getEnv('STARKWARE_HEALTH_ENDPOINT'),
+    responsePath: [],
+  },
 }
 
 export interface ExtendedConfig extends Config {
   delta: number
   deltaBlocks: number
   timeoutLimit: number
+  starkwareConfig: {
+    sequencerUrl: string
+    feederGatewayUrl: string
+    gatewayUrl: string
+    argentAccountAddr: string
+  }
 }
 
+const DEFAULT_STARKWARE_SEQUENCER_ENDPOINT = 'https://alpha-mainnet.starknet.io'
+const DEFAULT_STARKWARE_FEEDER_GATEWAY_URL = 'feeder_gateway'
+const DEFAULT_STARKWARE_GATEWAY_URL = 'gateway'
+const DEFAULT_STARKWARE_ARGENT_ACCOUNT_ADDR =
+  '0xd175dcf2fcf9d858d8d686826d56db7aeb29c3490110b4cfe0d8442944b828'
```

Comment:
Is this an account deployed to testnet or mainnet? 
How do you envision handling this for any network?

Reply:
this is an account I deployed to mainnet.  We can handle this for other networks by deploying to that network and setting an environment variable to overwrite the default.  The reason I've added a default is because EngOps often asks for this so that NOPs don't have to worry about it when running the EA.

Reply:
If that is the case, please add in-line comments documenting this.

Please also take this comment into account before you do that: https://github.com/smartcontractkit/external-adapters-js/pull/2039#discussion_r937584988

Reply:
todo:  Check to see if we still get 0 gas error with 0 address.